### PR TITLE
Feature: DHT11 Sensor Docs

### PR DESCRIPTION
**Pull request by tobiassiegrist**
Documentation of the DHT11 implementation.

This PR is tracked in tracked as ppmathis/fhnw-crowpi#13 and implemented in Pi4J/pi4j-example-crowpi#11

### CODE REVIEW

**Comment by tobiassiegrist** [comment]
@dependabot merge

File: content/german/components/humi-temp-sensor.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Temperatur- / Luftfeuchtigkeitssensor
+tags: ["gpio"]
+---
+
+## Funktionsweise
+
+Der im CrowPi benutzte DHT11 Temperatur- und Luftfeuchtigkeitssensor arbeitet für die Messung der Temperatur mit einem NTC-Temperatursensor.
+Dies ist ein temperaturabhängiger elektrischer Widerstand. Je nach Umgebungstemperatur leitet dieser den Strom besser oder schlechter. Diese
+Differenzen lassen sich messen und anhand von Referenzwertens kann die Temperatur ausgewertet werden. Das ist ein sehr einfaches Prinzip
```
Comment (ppmathis) [inline content/german/components/humi-temp-sensor.md:10]:
```suggestion
Differenzen lassen sich messen und anhand von Referenzwerten kann die Temperatur ausgewertet werden. Das ist ein sehr einfaches Prinzip,
```

File: content/german/components/humi-temp-sensor.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+welches häufig zum Einsatz kommt bei der Messung von Temperaturen. Für die Feuchtigkeit liegt zwischen zwei Elektroden ein spezielles
+Granulat. Je mehr Feuchte in diesem Granulat festgehalten wird, desto einfacher kann der Strom fliessen. Es ändert sich also auch der
```
Comment (ppmathis) [inline content/german/components/humi-temp-sensor.md:12]:
```suggestion
Granulat. Je mehr Feuchte in diesem Granulat festgehalten wird, desto einfacher kann der Strom fliessen. Es ändert sich also auch hier der
```

File: content/german/components/humi-temp-sensor.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+elektrische Widerstand. Die Übertragung der Messdaten erfolgt dann etwas speziell über einen einzelnen digitalen Ausgang am Sensor. Dieser
+wird in kurzen Pulsen im Mikrosekundenbereich angesteuert.
+
+Java mit Pi4J ist leider nicht in der Lage diese Pulse genügend schnell zu verarbeiten. Die Verzögerungen durch die vielen Schichten einer
+Applikation bis zur Hardware benötigen einfach zu viel Zeit. Deshalb wurde in diesem Beispiel ein Linux Treiber verwendet, welcher die
+Messwerte des Sensors in eine Datei schreibt. Diese wird von Java ausgelesen und so kann dennoch mit dem Sensor gearbeitet werden. Das ist
+keine optimale Lösung aber es ermöglicht immerhin die Arbeit mit dem Sensor innerhalb des CrowPi. Besser wäre eine separate Ansteuerung
+mittels eines Mikrocontrollers. Diese können aufgrund der einfachheit ihres Aufbaues viel schneller auf die eingehenden Impulse reagieren
```
Comment (ppmathis) [inline content/german/components/humi-temp-sensor.md:20]:
```suggestion
mittels eines Mikrocontrollers. Diese können aufgrund der Einfachheit ihres Aufbaus viel schneller auf die eingehenden Impulse reagieren
```

File: content/german/components/humi-temp-sensor.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+| Methode                   | Bemerkung                                               |
+|:--------------------------|:--------------------------------------------------------|
+| `double getTemperature()` | Gibt den letzten Temperaturmesswert in °C zurück.       |
+| `double getHumidity()`    | Gibt den letzten Luftfeuchtigkeitsmesswert in % zurück. |
+
+
+## Beispielapplikation
+
+Die sehr simple Beispielapplikation misst mithilfe eines `for-loops` einige Male die Temperatur und Luftfeuchtigkeit und gibt diese auf der
+Konsole aus. Verwendet werden dazu die Methoden `getHumidity()` und `getTemperature()` welche jeweils den aktuellen Messwert als `Double`
```
Comment (ppmathis) [inline content/german/components/humi-temp-sensor.md:54]:
```suggestion
Konsole aus. Verwendet werden dazu die Methoden `getHumidity()` und `getTemperature()` welche jeweils den aktuellen Messwert als `double`
```

**Review by ppmathis — CHANGES_REQUESTED** [review]
@tobiassiegrist LGTM aside from a few minor points, thank you for your work.

**Review by ppmathis — APPROVED** [review]

**Pull request merged**